### ISSUE TITLE
Make rich links accessible for Comment, Letter, Editorial designs

### ISSUE THREAD

**Issue by MarSavar** [guardian/dotcom-rendering#4927]
In these three designs, rich links have a dark grey background which fails the accessibility check when a rich link points to a comment article (orange on dark grey).

In order to make them accessible, we should use a lighter grey, as we do in other designs.

<img width="208" alt="image" src="https://user-images.githubusercontent.com/57295823/169059866-b1ab0ecd-cae7-4fe1-9031-d5afd3c58618.png">
